### PR TITLE
fix: argo-wf persistence configuration

### PR DESCRIPTION
**Pull request by bakhtin**
Signed-off-by: Artyom Bakhtin <a@bakhtin.net>

This PR fixes YAML indentation for `persistence` configuration. It was previously defined as a configuration option in `metricsConfig` section while it should be [one level higher](https://github.com/argoproj/argo/blob/f2ca045e1cad03d5ec7566ff7200fd8ca575ec5d/docs/workflow-controller-configmap.yaml#L125).

I also enabled Workflow Controller to access the secrets. It is required to properly configure `persistence` section.

Checklist:

* [X] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [X] I have signed the CLA and the build is green.
* [X] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by clementlecorre** [comment]
When I set up a secret for persistence, I have a permission error. 
I think adding `secrets` to the `workflow-controller-clusterrole.yaml` will solve my problem. 👍 

```
time="2020-03-24T14:21:31Z" level=warning msg="Failed to get secret 'my-postgres-config': secrets \"my-postgres-config\" is forbidden: User \"system:serviceaccount:my-namespace:argo\" cannot get resource \"secrets\" in API group \"\" in the namespace \"my-namespace\""
Error: secrets "my-postgres-config" is forbidden: User "system:serviceaccount:my-snamespace:argo" cannot get resource "secrets" in API group "" in the namespace "my-namespace"
secrets "my-postgres-config" is forbidden: User "system:serviceaccount:my-snamespace:argo" cannot get resource "secrets" in API group "" in the namespace "my-namespace"
Usage:
...
...
````

**Comment by stefansedich** [comment]
@bakhtin the secrets permission issue has been fixed in https://github.com/argoproj/argo-helm/pull/307

Do you want to rebase and just commit the indentation change as I think we still want to fix that up.

**Comment by bakhtin** [comment]
@stefansedich Rebased on master. Please, take a look

File: charts/argo/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "v2.7.6"
+appVersion: "v2.7.7"
```
Comment (stefansedich) [inline charts/argo/Chart.yaml:2]: No need to change the `appVersion`, the correct value here is still `v2.7.6`.

Comment (bakhtin) [reply charts/argo/Chart.yaml]: fixed

**Comment by stefansedich** [comment]
My bad @bakhtin you will have to bump the "chart version", forgot to mention that earlier, with that done we can merge this in!

**Comment by bakhtin** [comment]
> My bad @bakhtin you will have to bump the "chart version", forgot to mention that earlier, with that done we can merge this in!
> 
> For reference bump this one
> 
> https://github.com/argoproj/argo-helm/blob/e374150c3634b1f321db0e811d732580305ed068/charts/argo/Chart.yaml#L5

done

**Comment by bakhtin** [comment]
@stefansedich are we all set to merge this?

**Comment by stefansedich** [comment]
> @stefansedich are we all set to merge this?

For some reason the build never ran or reported back... looking into that now.

**Comment by stefansedich** [comment]
Ok finally @bakhtin sorry about that I got distracted the other night waiting for the build to finish and forgot to look into why it never reported status, looks like it never kicked off at all.

I made a small change to the indenting to get it to kick off and we should be good now!

**Pull request merged**